src/core/Akka.Cluster/ClusterDaemon.cs

```diff
@@ -1121,7 +1118,7 @@ protected override void PreStart()
 
             if (_seedNodes.IsEmpty)
             {
-                _log.Info("No seed-nodes configured, manual cluster join required");
+                _cluster.LogInfo("No seed-nodes configured, manual cluster join required");
```

Comment:
Made this change in about 45 places inside the `ClusterCoreDaemon` - allows the `akka.cluster.log-info` setting to toggle outputting these messages on and off and also appends the `SelfAddress` of the cluster to each of the inbound and outbound messages, which should improve the quality of the data we get from the logs. In the future the `InfoLogger` will also include the ID of the datacenter this node is participating in.